diff -burN udisks-2.0.92/configure.ac udisks-2.0.92.new/configure.ac
--- udisks-2.0.92/configure.ac	2013-01-06 17:45:52.000000000 -0600
+++ udisks-2.0.92.new/configure.ac	2013-03-09 16:28:16.312811489 -0600
@@ -145,7 +145,7 @@
 AC_SUBST(LIBATASMART_LIBS)

 PKG_CHECK_MODULES(LIBSYSTEMD_LOGIN,
-                  [libsystemd-login >= 44 libsystemd-daemon],
+                  [libsystemd-login >= 44 libsystemd-daemon libsystemd-id128],
                   have_libsystemd_login=yes,
                   have_libsystemd_login=no)
 AM_CONDITIONAL(HAVE_LIBSYSTEMD_LOGIN, test x$have_libsystemd_login = xyes)
diff -burN udisks-2.0.92/tools/Makefile.am udisks-2.0.92.new/tools/Makefile.am
--- udisks-2.0.92/tools/Makefile.am	2012-10-07 11:20:42.000000000 -0500
+++ udisks-2.0.92.new/tools/Makefile.am	2013-03-09 23:10:02.655732140 -0600
@@ -34,6 +34,7 @@
	$(POLKIT_GOBJECT_1_LIBS)			\
	$(POLKIT_AGENT_1_LIBS)				\
	$(top_builddir)/udisks/libudisks2.la		\
+	$(LIBSYSTEMD_LOGIN_LIBS)			\
	$(NULL)

 # ----------------------------------------------------------------------------------------------------
